TrustKit/TSKPinningValidator.h

```diff
@@ -80,26 +105,24 @@ typedef NS_ENUM(NSInteger, TSKTrustDecision)
  
  @exception NSException Thrown when TrustKit has not been initialized with a pinning policy.
  */
-+ (TSKTrustDecision) evaluateTrust:(SecTrustRef _Nonnull)serverTrust forHostname:(NSString * _Nonnull)serverHostname;
+- (TSKTrustDecision)evaluateTrust:(SecTrustRef _Nonnull)serverTrust forHostname:(NSString * _Nonnull)serverHostname;
 
 
 /**
  Helper method for handling authentication challenges received within a `NSURLSessionDelegate`, `NSURLSessionTaskDelegate` or `WKNavigationDelegate`.
  
  This method will evaluate the server trust within the authentication challenge against the global SSL pinning policy previously configured, and then call the `completionHandler` with the corresponding `disposition` and `credential`. For example, this method can be leveraged in a `WKNavigationDelegate` challenge handler method:
-     
-     - (void)webView:(WKWebView *)webView
-     didReceiveAuthenticationChallenge:(NSURLAuthenticationChallenge *)challenge
-     completionHandler:(void (^)(NSURLSessionAuthChallengeDisposition disposition,
-     NSURLCredential *credential))completionHandler
-     {
-         if (![TSKPinningValidator handleChallenge:challenge completionHandler:completionHandler]) 
-         {
-             // TrustKit did not handle this challenge: perhaps it was not for server trust
-             // or the domain was not pinned. Fall back to the default behavior
-             completionHandler(NSURLSessionAuthChallengePerformDefaultHandling, nil);
-         }
-     }
+
+    - (void)webView:(WKWebView *)webView
+    didReceiveAuthenticationChallenge:(NSURLAuthenticationChallenge *)challenge
+    completionHandler:(void (^)(NSURLSessionAuthChallengeDisposition disposition,
+                                NSURLCredential *credential))completionHandler
+    {
+        if ([challenge.protectionSpace.authenticationMethod isEqualToString:NSURLAuthenticationMethodServerTrust])
+        {
+            [TSKPinningValidator handleChallenge:challenge completionHandler:completionHandler];
+        }
+    }
```

Comment:
Right this comment doesn't look right. I don't know why I changed it.

I checked my implementation in Yahoo Finance and it's correct so I just don't recall this... good catch.